lib/travis/build/script/matlab.rb

```diff
@@ -32,12 +32,13 @@ def setup
           sh.fold 'matlab_install' do
```

Comment:
Should we change the first echo to something better then what gets spit out of matlab-deps? Maybe something like "Installing MATLAB"